src/encoded/commands/deploy.py

```diff
@@ -179,7 +183,7 @@ def _tag_ec2_instance(
         tags_dict['elastic_ip_swtich_datatime'] = 'na'
         if cluster_master:
             tags_dict['elasticsearch_head'] += 'yes'
```

Comment:
This line (185) should be = 'yes' not += 'yes'

Reply:
yes, this is fixed along with a couple other bugs in the current pr.  There is also a read me now.

Reply:
Looks good!